pkg/agent/cniserver/interface_configuration_windows.go

```diff
@@ -338,6 +338,16 @@ func (ic *ifConfigurator) validateContainerPeerInterface(interfaces []*current.I
 		containerVeth.name)
 }
 
+// getInterceptedInterfaces is not supported on Windows.
+func (ic *ifConfigurator) getInterceptedInterfaces(
+	sandbox string,
+	containerNetNS string,
+	containerIFDev string,
+) (*current.Interface, *current.Interface, error) {
+	klog.Info("getInterceptedInterfaces is not supported on Windows, return immediately")
```

Comment:
The code can only be run if user configure it mistakenly, so instead of letting the code run without error and may panic somewhere, should we return error to point out this is not supported explicitly, like https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cadvisor/cadvisor_unsupported.go?

Reply:
Got it, will change.